examples/libs/img_font/lv_example_img_font_1.c

```diff
@@ -0,0 +1,65 @@
+#include "../../lv_examples.h"
+#include <stdio.h>
+
+#if LV_BUILD_EXAMPLES
+#if LV_USE_IMG_FONT
+
+LV_IMG_DECLARE(emoji_F617);
+static bool get_img_path(lv_img_font_t * handle, char * path, uint16_t len, uint32_t unicode, uint32_t unicode_next)
+{
+    LV_UNUSED(unicode_next);
+
+    if(unicode == 0xF617) {
+        memcpy((void *)path, &emoji_F617, sizeof(lv_img_dsc_t));
+    }
+    else {
+        snprintf(path, len, "%s/%04X.%s", "./lvgl/examples/assets/emoji", unicode, "png");
+    }
+    path[len - 1 ] = '\0';
+
+    return true;
+}
+
+/**
+ * draw img in label or span obj
+ */
+void lv_example_img_font_1(void)
+{
+    static lv_img_font_t img_font;
+    img_font.font = NULL;
```

Comment:
What about an `lv_img_font_init(height, get_img_cb)` function?

Reply:
Is it better to use ` lv_font_t * lv_imgfont_create(height, get_img_cb) ` and keep it in pairs with `void lv_imgfont_destroy(lv_font_t * font)`

Reply:
Sounds great!